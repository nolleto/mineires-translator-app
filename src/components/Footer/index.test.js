import Footer from './index';
import React from 'react';
import { shallow } from 'enzyme';

describe('Footer component', () => {
  it('matches snapshot', () => {
    const snapshot = shallow(<Footer />);

    expect(snapshot).toMatchSnapshot();
  });
});
