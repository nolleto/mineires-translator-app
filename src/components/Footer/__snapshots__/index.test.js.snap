// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component matches snapshot 1`] = `
<footer
  className="footer"
>
  <div
    className="footer__line"
  />
  <p>
    Made with 
    <span
      className="footer--red"
    >
      ❤︎
    </span>
     by 
    <a
      href="https://twitter.com/FelipeNolleto"
    >
      nolleto
    </a>
    . Find this on 
    <a
      href="https://github.com/nolleto/mineires-translator-app"
    >
      github
    </a>
  </p>
</footer>
`;
